torch/csrc/jit/compiler/tests/llvm_test.cc

```diff
@@ -1,18 +1,19 @@
 #include "torch/csrc/jit/compiler/include/ir.h"
 #include "torch/csrc/jit/compiler/include/llvm_codegen.h"
 
+#include <gmock/gmock.h>
```

Comment:
This looks like a new dependency that needs to be added. This does not build even with gmock installed on my machine

Reply:
I'll just remove gmock.  It's supposed to be part of gtest, but I'm using it for something trivial anyways so no big deal to omit it.  I still don't understand why it works for me (tm) and not for you, but it's not worth tracking down :)